tests/ui/by_value_not_supported.stderr

```diff
@@ -16,7 +16,7 @@ error: using C++ string by value is not supported
 6 |         s: CxxString,
   |         ^^^^^^^^^^^^
 
-error: needs a cxx::ExternType impl in order to be used as a field of `S`
+error: needs a cxx::ExternType impl in order to be used as a field of `::S`
```

Comment:
I think this was more correct before. The type being discussed by the error message is not `::S` in Rust. It's actually `crate::ffi::S` but it would suffice to call it `S` for this message. `::S` is the C++ name but doesn't feel appropriate to use for this message.

Reply:
OK, agreed, will do